addons/api/mirage/config.js

```diff
@@ -682,13 +682,24 @@ function routes() {
       const id = idMethod.split(':')[0];
       const method = idMethod.split(':')[1];
       const worker = workers.find(id);
-      let updatedAttrs = {};
+      let updatedAttrs = {
+        version: attrs.version,
+        apiTags: worker.api_tags,
+      };
+
+      if (method === 'remove-worker-tags') {
+        const key = Object.keys(attrs.apiTags)[0];
+        const value = attrs.apiTags[key][0];
+        updatedAttrs.apiTags[key] = updatedAttrs.apiTags[key].filter((tag) => {
+          return tag !== value;
+        });
+        if (updatedAttrs.apiTags[key].length === 0) {
+          delete updatedAttrs.apiTags[key];
```

Comment:
should this be handled in the serializer?

Reply:
Thanks for the suggestion. I forgot we had mirage serializers. I have updated this. Please let me know if what I added is correct.